### ISSUE TITLE
SADeprecationWarning: Implicit coercion of SELECT and textual SELECT constructs into FROM clauses is deprecated

### ISSUE THREAD

**Issue by Moult** [BrickSchema/py-brickschema#117]
It seems this print statement is triggering deprecated warnings (caught from Bonsai test suite).

```
  File "/home/dion/.config/blender/4.3/extensions/.local/lib/python3.11/site-packages/brickschema/persistent.py", line 289, in new_changeset
    print(f"{c.identifier} has {len(c)} triples")
                                ^^^^^^
  File "/home/dion/.config/blender/4.3/extensions/.local/lib/python3.11/site-packages/rdflib/graph.py", line 719, in __len__
    return self.__store.__len__(context=self)  # type: ignore[call-arg]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dion/.config/blender/4.3/extensions/.local/lib/python3.11/site-packages/rdflib_sqlalchemy/store.py", line 208, in __len__
    q = union_select(selects, distinct=True, select_type=COUNT_SELECT)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dion/.config/blender/4.3/extensions/.local/lib/python3.11/site-packages/rdflib_sqlalchemy/sql.py", line 65, in union_select
    select_clause = expression.select(*[functions.count().label('aCount')]).select_from(
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<string>", line 2, in select_from
  File "/home/dion/.config/blender/4.3/extensions/.local/lib/python3.11/site-packages/sqlalchemy/sql/base.py", line 279, in _generative
    x = fn(self, *args, **kw)
        ^^^^^^^^^^^^^^^^^^^^^
  File "/home/dion/.config/blender/4.3/extensions/.local/lib/python3.11/site-packages/sqlalchemy/sql/selectable.py", line 6283, in select_from
    self._from_obj += tuple(
                      ^^^^^^
  File "/home/dion/.config/blender/4.3/extensions/.local/lib/python3.11/site-packages/sqlalchemy/sql/selectable.py", line 6284, in <genexpr>
    coercions.expect(
  File "/home/dion/.config/blender/4.3/extensions/.local/lib/python3.11/site-packages/sqlalchemy/sql/coercions.py", line 424, in expect
    return impl._implicit_coercions(
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dion/.config/blender/4.3/extensions/.local/lib/python3.11/site-packages/sqlalchemy/sql/coercions.py", line 1281, in _implicit_coercions
    util.warn_deprecated(
  File "/home/dion/.config/blender/4.3/extensions/.local/lib/python3.11/site-packages/sqlalchemy/util/deprecations.py", line 58, in warn_deprecated
    _warn_with_version(
  File "/home/dion/.config/blender/4.3/extensions/.local/lib/python3.11/site-packages/sqlalchemy/util/deprecations.py", line 52, in _warn_with_version
    _warnings_warn(warn, stacklevel=stacklevel + 1)
  File "/home/dion/.config/blender/4.3/extensions/.local/lib/python3.11/site-packages/sqlalchemy/util/langhelpers.py", line 1898, in _warnings_warn
    warnings.warn(message, stacklevel=stacklevel + 1)
sqlalchemy.exc.SADeprecationWarning: Implicit coercion of SELECT and textual SELECT constructs into FROM clauses is deprecated; please call .subquery() on any Core select or ORM Query object in order to produce a subquery object.
Location: /home/dion/drive/blender/blender-4.3.2-linux-x64/4.3/scripts/modules/bpy/ops.py:109
```
